client/components/jetpack/card/jetpack-product-slide-out-card/style.scss

```diff
@@ -0,0 +1,147 @@
+$jetpack-product-slide-out-card-color: var( --studio-wordpress-blue-30 );
+
+.jetpack-product-slide-out-card {
+	box-sizing: border-box;
+
+	background: var( --color-surface );
+	border: 1px solid var( --color-border-subtle );
+	box-shadow: inset 0 4px 8px rgba( 0, 0, 0, 0.06 );
+}
+
+.jetpack-product-slide-out-card__header {
+	position: relative;
+
+	padding: 24px 16px 12px;
+
+	text-align: center;
+}
+
+.jetpack-product-slide-out-card .product-icon {
+	width: 32px;
+
+	margin-right: 16px;
+}
+
+.jetpack-product-slide-out-card__product-name {
```

Comment:
Figma design has the product name color as #00131C (--studio-blue-100)

Reply:
You are right. But I wonder if this a mistake or it is intentional because I can see that the name in the product card component also has the same color.

Reply:
@eeeeevon13 do we want to use `#00131C (--studio-blue-100)` for product names in both the slide out card and the product card?